apps/io-wallet-user-func/src/app/config.ts

```diff
@@ -246,21 +254,27 @@ export const getAzureConfigFromEnvironment: RE.ReaderEither<
       appInsights,
       cosmos,
       entityConfigurationStorageContainerName,
+      pidIssuerRevokeApiQueueName,
       revocationQueueName,
       storageAccountConnectionString,
     }) => ({
       appInsights,
       cosmos,
-      queue: {
-        walletInstanceRevocation: {
-          connectionString: storageAccountConnectionString,
-          name: revocationQueueName,
-        },
-      },
       storage: {
         entityConfiguration: {
           containerName: entityConfigurationStorageContainerName,
         },
+        walletInstances: {
+          connectionString: storageAccountConnectionString,
+          queues: {
+            pidIssuerRevokeApi: {
+              name: pidIssuerRevokeApiQueueName,
+            },
```

Comment:
Why inside the Wallet Instances object?

Reply:
because this is triggered during the wallet instance revocation flow. The key point however is that we have a single storage account containing three (wallet instance-related) queues and one queue dedicated to entity configuration. We can use another name